Makefile

```diff
@@ -3,7 +3,7 @@
 install:
 	npm install
 
-spectral:
+spectral: install
```

Comment:
Why is this needed? I thought librarian does a `make install` as part of the `init` step.